Source/WebKit/Platform/spi/Cocoa/AccessibilitySupportSPI.h

```diff
@@ -29,7 +29,7 @@
 #include <AccessibilitySupport.h>
 #endif
 
-#if !USE(APPLE_INTERNAL_SDK) || !HAVE(APP_SPECIFIC_ACCESSIBILITY_SETTINGS)
```

Comment:
Did you remove it from PlatformHave.h in an earlier change? I don't see it being removed here.

Reply:
Good catch! I managed to delete all the uses of it but not the original definition.